org.eclipse.xtext.ui/deprecated/org/eclipse/xtext/ui/editor/contentassist/antlr/AbstractContentAssistParser.java

```diff
@@ -7,19 +7,38 @@
  *******************************************************************************/
 package org.eclipse.xtext.ui.editor.contentassist.antlr;
 
+import org.antlr.runtime.CharStream;
+import org.antlr.runtime.TokenSource;
 import org.eclipse.xtext.AbstractElement;
+import org.eclipse.xtext.ide.LexerIdeBindings;
 import org.eclipse.xtext.ide.editor.contentassist.antlr.BaseContentAssistParser;
 import org.eclipse.xtext.ide.editor.contentassist.antlr.EofListener;
 import org.eclipse.xtext.ide.editor.contentassist.antlr.internal.BaseInternalContentAssistParser;
 import org.eclipse.xtext.ui.editor.contentassist.antlr.internal.AbstractInternalContentAssistParser;
+import org.eclipse.xtext.ui.editor.contentassist.antlr.internal.Lexer;
+
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import com.google.inject.name.Named;
 
 /**
- * This class if effectively deprecated and clients should use the equivalent from
- * the ide package instead.
+ * This class if effectively deprecated and clients should use the equivalent from the ide package instead.
  * 
  * @author Sebastian Zarnekow - Initial contribution and API
  */
-public abstract class AbstractContentAssistParser extends BaseContentAssistParser<FollowElement, LookAheadTerminal, AbstractInternalContentAssistParser> implements IContentAssistParser {
+public abstract class AbstractContentAssistParser extends
+		BaseContentAssistParser<FollowElement, LookAheadTerminal, AbstractInternalContentAssistParser> implements IContentAssistParser {
+
+	@Inject
+	@Named(LexerIdeBindings.CONTENT_ASSIST)
```

Comment:
should that use the "old" constant?

Reply:
On master it was already LexerIdeBindings.CONTENT_ASSIST, so I have no worries.